tests/run/matrix_multiplier.pyx

```diff
@@ -22,9 +22,11 @@ ExtMatMult("ExtMatMult('ExtMatMult(1) @ ExtMatMult(2)') @ ExtMatMult(2)")
 >>> x @ y
 Traceback (most recent call last):
 TypeError: unsupported operand type(s) for @: 'int' and 'int'
->>> x @= y
+
+PyPy exception message has '@' rather than '@='
+>>> x @= y  # doctest: +ELLIPSIS
 Traceback (most recent call last):
-TypeError: unsupported operand type(s) for @=: 'int' and 'int'
+TypeError: unsupported operand type(s) for @...: 'int' and 'int'
```

Comment:
@mattip Not a big issue, but arguable somewhere that PyPy's error reporting could be improved

Reply:
Thanks. I opened https://foss.heptapod.net/pypy/pypy/-/issues/3605